src/workerd/api/container.h

```diff
@@ -21,7 +23,11 @@ class Fetcher;
 // etc.
 class Container: public jsg::Object {
  public:
+  // Constructor for RPC-based containers (production)
   Container(rpc::Container::Client rpcClient, bool running);
+  
+  // Constructor for Docker-based containers (local development)
+  Container(kj::String containerId, kj::String imageTag, io::DockerClient& dockerClient);
```

Comment:
I think Kention mentioned that the generated interface would be the same (even if there is no "transport" when the the server runs in the same process)